matcher/matcher.py

```diff
@@ -0,0 +1,109 @@
+import jmespath
+import json
+import re
+
+
+
+
+def apply_operator(j_val, op, val):
```

Comment:
Calling the arguments `j_val` and `val` may be a bit confusing. Maybe just `val1` and `val2`?